subiquity/common/errorreport.py

```diff
@@ -140,6 +141,9 @@ def _bg_add_info():
                 self.pr.add_hooks_info(None)
                 apport.hookutils.attach_hardware(self.pr)
             self.pr['Syslog'] = apport.hookutils.recent_syslog(re.compile('.'))
+            snap_name = os.environ.get('SNAP_NAME', None)
+            if bool(snap_name):
+                self.add_tags([snap_name])
```

Comment:
suggestion:
```python
with contextlib.suppress(KeyError):
    self.add_tags([os.environ["SNAP_NAME"])
```

Reply:
this would also suppress KeyError if self.add_tags raises one, so in hindsight, it may not be a great idea.